AirCasting/SDSync/ViewsFlow/BackendSyncCompleted/BackendSyncCompletedViewModel.swift

```diff
@@ -0,0 +1,40 @@
+// Created by Lunar on 02/12/2021.
+//
+
+import Foundation
+
+enum ProceedToSyncView {
+    case restart
+    case bluetooth
+}
+
+protocol BackendSyncCompletedViewModel: ObservableObject {
+    var presentRestartNextScreen: Bool { get set }
+    var presentBTNextScreen: Bool { get set }
+    var urlProvider: BaseURLProvider { get }
```

Comment:
I assume this is only used for dependency injection reasons - please leave a comment